convert2rhel/pkghandler.py

```diff
@@ -849,6 +851,11 @@ def remove_pkgs_unless_from_redhat(pkgs_to_remove, backup=True):
     loggerinst.warning("Removing the following %s packages:\n" % str(len(pkgs_to_remove)))
     print_pkg_info(pkgs_to_remove)
     loggerinst.info("\n")
+
+    # Need to generate backed up path of repo files since hash is used to avoid
+    # conflicts during backup
+    backedup_reposdir = os.path.join(utils.BACKUP_DIR, hashlib.md5(DEFAULT_YUM_REPOFILE_DIR.encode()).hexdigest())
```

Comment:
Not sure if I followed on why this needs to be here

Reply:
This function is used to remove packages from other sources as well, but I don't see it being used during the backup of the variables/yum repos. 

Ideally, who should take care of that hash is the RestorableFile class. Also, this function got moved away in my last PR: https://github.com/oamg/convert2rhel/pull/1091

Reply:
@r0x0d We need to have access to the backed up repofiles when we are trying to backup centos-release file (with this file I had issues). Before porting to the resotrablefile, just backup path was used and there were available the repofiles. But now, when we copy the .repo files to the directory named as the hash of the original path, the repofiles are inaccessible. 

So I've chosen this strategy of building the path here since it's probably just this one case.

Reply:
Ah, I see. So we might receive this from the parameter in the function rather than hardcoding it in the function body. 

We can make this function being more generic if we receive the reposdir, and whoever needs this information, we can pass this down to the function.